platform/installation/offline-installations.mdx

```diff
@@ -21,6 +21,34 @@ To solve not being able ot pull images from Docker hub, you can
 [use a private image registry](#offline-images) to host your own copies of the vCluster Platform image that
 your cluster can reach.
 
+### Prerequisites
+
+- A running air-gapped Kubernetes cluster `v1.26.x`, `v1.27.x`, or `v1.28.x`
+- On installation computer:
+    - `kubectl` (check via `kubectl version`)
+    - `helm` (check with `helm version`)
+    - `docker` (check with `docker version`)
+    - A `kube-context` with admin access to this Kubernetes cluster (check with `kubectl auth can-i create clusterrole -A`)
+    - [Optional] An offline license key for vCluster.Pro (provided by Loft)
+- [Optional] A private docker registry that the installer computer and the air-gapped Kubernetes cluster can access
+
+### Resource requirements
+
+- vCluster.Pro manager pod resource requirements:
+    - Requests
+        - memory: `256Mi`
+        - cpu: `200m`
+    - Limits
+        - memory: `4Gi`
+        - cpu: `2`
+
+- Open network ports:
+    > Since loft installs webhooks and api server extensions into the cluster, the kubernetes master needs to be able to communicate with the loft pods. In private GKE clusters the kubernetes master and nodes for example are not in the same subnetwork, and cannot communicate directly with each other on every port. Hence, you need to ensure that there is a firewall rule that allows incoming traffic from the kubernetes master network to the tcp ports:
```

Comment:
Instead of using "loft", we want to use moving towards "vCluster Platform" if it makes sense. 

Can we also capitalize Kubernetes?

Reply:
Updated 👍